### PR TITLE
always collect selenium logs 

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 apypie
 pytest-base-url
+pytest-html
```
Comment (ekohl) [inline requirements.txt:3]: Why is this needed? Is it for https://github.com/theforeman/forklift/pull/1262?

Comment (evgeni) [reply requirements.txt:3]: sorta. pytest-selenium does require pytest-html anyways (and without using the html report, no logs are fetched :roll_eyes:), but it felt weird to call `--html` without an explicit dependency

**Review by ekohl — APPROVED** [review]
